tf/scripts/frontend_user_data.sh

```
@@ -16,5 +16,12 @@ sudo docker build -t lti-frontend .
 # Ejecutar el contenedor Docker
 sudo docker run -d -p 3000:3000 lti-frontend
 
+# Install Datadog Agent with EU region and custom tags
+DD_API_KEY=${datadog_api_key} \
```

Shellcheck (0.11.0)

[warning] datadog_api_key is referenced but not assigned.

(SC2154)